.circleci/deploy.sh

```diff
@@ -85,19 +88,29 @@ done
 # Copy "submitted_data" to "cimr-root" bucket (private)
 aws s3 sync submitted_data/ s3://cimr-root/${PR_STR}/ --exclude "request.handled"
 
+# Update the repo
+git pull
+
 # Move submitted YAML files to "processed/" sub-dir
 mkdir -p processed/${PR_STR}/
-git mv -k submitted/*.yml submitted/*.yaml processed/${PR_STR}/
+SUBMITTED_FILES=$(curl -s ${GH_PR_API} | jq -r '.[].filename' | grep "^submitted/") || true
+if [ -z "${SUBMITTED_FILES}" ]; then
```

Comment:
if there are no ${SUBMITTED_FILES}, we shouldn't make processed/${PR_STR}/ dir, no?

Reply:
Good point.